README.md

```diff
@@ -307,6 +308,7 @@ The default values for each rule are described below.
 | Rule                        | Default |
 | --------------------------- | --------|
 | no_request_body_content     | error   |
+| no_request_body_name        | error   |
```

Comment:
Might want to make this a `warning`, since it only affects SDK generation.

Reply:
Whoops, I did make it a warning in the code. Just documented it incorrectly. I'll fix that